exporter/clickhouseexporter/config.go

```diff
@@ -145,6 +170,38 @@ func (cfg *Config) shouldCreateSchema() bool {
 	return cfg.CreateSchema
 }
 
+func (cfg *Config) buildMetricTableNames() {
+	tableName := defaultMetricTableName
+
+	if len(cfg.MetricsTableName) != 0 && !cfg.areMetricTableNamesSet() {
+		tableName = cfg.MetricsTableName
+	}
+
+	if len(cfg.MetricsTables.Gauge.Name) == 0 {
```

Comment:
```suggestion
	if cfg.MetricsTables.Gauge.Name == "" {
```
Just a nits :)

Reply:
Oh I saw this too in a few places. Yes the compiler handles it the same. However...

While reading it, I kept asking myself "what type is this variable? is it a slice"? I agree it's more preference than performance, but I prefer the `== ""` since then I know it's a string. The rest of the code seems to prefer `== ""` too.

If checking a pointer type then a nil check would be better anyway. Ultimately I don't mind, but the codebase *would* prefer the `== ""` for consistency. Your choice 🤷